Makefile

```diff
@@ -131,7 +131,8 @@ fmt-check:
 
 .PHONY: test
 test:
-	$(GO) test $(PACKAGES)
+	$(GO) build # test if go build succeed without sqlite support
```

Comment:
Why is this under `test`?

Also, building without the sqlite tag already runs in CI (`make build`), so why is doing it again necessary?

Reply:
`make build` will always add `-tags=sqlite`. This is a test for building without sqlite. Since all test now always use `sqlite`.

Reply:
Okay, I see why building again, without the sqlite tag, is necessary. However, could we please not put this in the `test` make target, for the sake of keeping our make targets straightforward? One option would be to just run `make build` an additional time in CI (without the sqlite tag).

Reply:
Or Maybe a new section named `comiple-test` or `build-test`?